src/scheduler/check.cpp

```diff
@@ -1192,13 +1192,19 @@ check_avail_resources(schd_resource *reslist, resource_req *reqlist,
 				if (!compare_non_consumable(res, resreq)) {
 					fail = 1;
 					if (err != NULL) {
+						char *requested;
```

Comment:
Can you make this a `const char *`? Eventually I'd like `res_to_str_r` return a `const char *`,